src/pybind/mgr/requirements.txt

```diff
@@ -0,0 +1 @@
+pytest-cov==2.7.1
```

Comment:
Thanks for removing those! Does this work without pytest explicitly set?

Reply:
Yes it actually does :)